lib/extensions/brakeman_fingerprint_patch.rb

```diff
@@ -77,8 +77,6 @@ def fingerprint
   def to_hash(absolute_paths: true)
     super.tap do |h|
       h[:file] = (absolute_paths ? self.file.absolute : file_string)
-      h[:file_rel] = self.file.relative
-      h[:file_abs] = self.file.absolute
```

Comment:
I had these originally for completeness, but then they were being dumped into the brakeman.ignore file, which is not necessary, so I'm removing them here. There is no real loss of functionality.